spec/oncocore_icd10_vs.txt

```diff
@@ -2549,8 +2550,11 @@ ICD10CM#D4989       "Neoplasm of unspecified behavior of other specified sites"
 ICD10CM#D499       "Neoplasm of unspecified behavior of unspecified site"
 
 
-ValueSet:   SecondaryCancerConditionVS
-Description:  "Types of secondary malignant neoplastic disease. Includes SNOMED CT disorder hierarchy codes for secondard malignant neoplastic disease, as well as ICD-10-CM codes for malignant neoplasms and neoplasms of uncertain or unspecified behavior. If body site is not precoordinated (implied by the code), it shall be separately specified. For ICD-10-CM coding of the secondary disease, the disorder code should always be SCT#128462008 (Secondary malignant neoplastic disease), and histology/morphology/behavior code and the topology code (body site) and should be specified separately in the HistologyMorphologyBehavior extension and BodySite, respectively."
+ValueSet:   SecondaryCancerDisorderVS
+Description:  "Types of secondary malignant neoplastic disease, coded in SNOMED CT, ICD-10-CM, or ICD-O-3. 
+* SNOMED CT coding: Use a code from the disorder hierarchy under secondary malignant neoplastic disease (SNOMED-CT 128462008). 
+* ICD-10-CM coding: Use one of the codes given in this value set representing secondary malignant neoplasms and neoplasms of uncertain or unspecified behavior. If body site is not precoordinated (implied by the code), it should be specified separately using the body location. 
```

Comment:
This value set should not include neoplasms of uncertain or unspecified behavior. These are assumed to be primary in ICD-10-CM. If you don't know whether it's malignant, you certainly can't tell if it's secondary.